conda/environments/cudf_dev_cuda10.0.yml

```diff
@@ -41,8 +41,9 @@ dependencies:
   - distributed>=2.15.0
   - streamz
   - dlpack
-  - arrow-cpp=0.15.0
-  - boost-cpp
+  - arrow-cpp=0.17.1
+  - arrow-cpp-proc=*=cuda
```

Comment:
Here we use `=*=` below we use ` * ` and in rapidasi/integration#45 we use ` * `, as a matter of style can we use a consistent definition?

Reply:
Made the change